Validation/CTPPS/python/year_2016/ctppsOpticalFunctionsESSource_cfi.py

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
this file looks like a duplicate of what's already added in RecoCTPPS/Configuration/python/recoCTPPS_cff.py 
To avoid duplication, please address the comment in recoCTPPS_cff.py

Reply:
Duplicate files removed.